core/src/main/java/org/sonarsource/sonarlint/core/ConnectedSonarLintEngineImpl.java

```diff
@@ -237,6 +238,13 @@ public void updateModule(ServerConfiguration serverConfig, String moduleKey) {
       changeState(State.UPDATING);
       connectedContainer.startComponents();
       connectedContainer.updateModule(moduleKey);
+      Set<String> repoKeysFromPlugins = globalContainer.getRepoKeysFromPlugins();
+      Set<String> activeRepoKeys = globalContainer.getRepoKeysInQP(moduleKey);
+      activeRepoKeys.removeAll(repoKeysFromPlugins);
+      System.out.println("KEYS NOT IN PLUGINS:");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS106)